salt/config/__init__.py

```diff
@@ -919,6 +919,14 @@ def _gather_buffer_space():
 
     # Minion data cache driver (one of satl.cache.* modules)
     'cache': str,
+    # Enables a fast in-memory cache booster and sets the expiration time.
+    'memcache_expire_seconds': int,
```

Comment:
It's just about keeping some of `minion_data_cache` data in-memory to don't fetch them from the cache backend again and again. No any additional cache logic is here. With and without it the business logic has kept the same.